### DISCUSSION TITLE
Polylookup - Issue with selected values being lost after record saved for new record

### DISCUSSION THREAD

**Discussion by cyberdmt** [khoait/DCE.PCF#98] [General]
Hi Khoa,

Thanks for creating this great pcf control.

We have been experiencing this issue - where the selected values in the PolyLookup linked control is lost on save.
The Outputselectedfield is being set appropriately - to another text field on the form.
After saving the record, all works fine.

We have re-tested a form where PolyLookup was used in this same way at another site (which was previously working fine), and it is also behaving like this.

Thanks for your help,
Dominic

**Comment by khoait** [comment]
hi @cyberdmt , for create form, there're some steps you need to do in order for it to work. please refer this wiki

https://github.com/khoait/DCE.PCF/wiki/PolyLookup#output-selected-items-as-a-json-array
